.github/CONTRIBUTING.md

```diff
@@ -242,31 +279,28 @@ This prevents nesting levels from getting deeper then they need to be.
 
 * Information that players could use to metagame (that is, to identify round information and/or antagonist type via information that would not be available to them in character) should be kept as administrator only.
 
-* It is recommended as well you do not expose information about the players - even something as simple as the number of people who have readied up at the start of the round can and has been used to try to identify the round type.
-
-* Where you have code that can cause large-scale modification and *FUN*, make sure you start it out locked behind one of the default admin roles - use common sense to determine which role fits the level of damage a function could do.
+* Where you have code that can cause large-scale modification and *FUN*, make sure you start it out locked behind one of the proper admin permissions - ask if you're not sure.
```

Comment:
Why are you taking out the previous statement?

Reply:
Because the current code is already breaking that. Who will be playing or observing is public information.

Reply:
Though I agree, I should include something that mentions just IPs and CIDs.

Reply:
sure but it's still bad to add more

Reply:
For private information, I agree, for the ready players and such, it's heavily context dependent for our server so putting it here wouldn't really be wise.

Reply:
It's not though, this is an anti powergamer standard. It makes sense to have this in the contribution guidelines still

Reply:
It does not for our server.